tests/unit/dataset/loader/test_burst_gpt_trace.py

```
@@ -0,0 +1,230 @@
+
+    def test_non_csv_file_returns_false(self, tmp_path: Path) -> None:
+        path = tmp_path / "data.jsonl"
+        path.write_text('{"input_length": 10}\n')
+        assert BurstGPTTraceDatasetLoader.can_load(filename=path) is False
+
+    def test_none_filename_returns_false(self) -> None:
+        assert BurstGPTTraceDatasetLoader.can_load(filename=None) is False
+
+    def test_nonexistent_file_returns_false(self) -> None:
+        assert (
+            BurstGPTTraceDatasetLoader.can_load(filename="/tmp/does_not_exist.csv")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/does_not_exist.csv"

(S108)